beets/importer.py

```diff
@@ -547,9 +547,22 @@ def _emit_imported(self, lib):
         plugins.send('album_imported', lib=lib, album=self.album)
 
     def emit_created(self, session):
-        """Send the `import_task_created` event for this task.
+        """Send the `import_task_created` event for this task and return a list
+        of tasks which might be returned by plugins.
         """
-        plugins.send('import_task_created', session=session, task=self)
+        tasks = plugins.send('import_task_created', session=session, task=self)
+        if not tasks:
+            tasks = [self]
+        else:
+            # The plugins gave us a list of lists of task. Flatten it.
+            flat_tasks = []
+            for inner in tasks:
+                if isinstance(inner, list):
```

Comment:
We could just require plugins to return lists instead of individual objects—that might be a bit more predictable.